ci/sge/Dockerfile-master

```diff
@@ -9,7 +9,9 @@ RUN curl -o miniconda.sh https://repo.continuum.io/miniconda/Miniconda3-latest-L
     /opt/anaconda/bin/conda clean -tipy && \
     rm -f miniconda.sh
 ENV PATH /opt/anaconda/bin:$PATH
-RUN conda install -c conda-forge dask distributed pytest && conda clean -tipy
+ARG TRAVIS_PYTHON_VERSION
+ENV TRAVIS_PYTHON_VERSION=$TRAVIS_PYTHON_VERSION
```

Comment:
Is the ENV line really needed, doesn't ARG suffice?

Reply:
ENV allows the run master/slave scripts access to TRAVIS_PYTHON_VERSION for the blocking script below.